tox.ini

```diff
@@ -49,7 +49,7 @@ commands =
     codespell . --skip .git --skip .tox --skip build --skip lib --skip venv* \
       --skip .mypy_cache --skip icon.svg
     # pflake8 wrapper supports config from pyproject.toml
-    pflake8 {[vars]all_path}
+    pflake8 {[vars]src_path} {[vars]lib_path}
```

Comment:
I think this should stay `{[vars]all_path}`.
What errors were you seeing?

Reply:
after you've run integration tests, the tester charms contain (for example) traefik_route.py, which fails linting.
So I guess it depends on the order in which you run the tests

we could selectively exclude all non-traefik charm libs from the tester charms, but that felt like too much work.

Reply:
(or we could fix linting in traefik_route, but I guess the general issue of linting third party charm libs remains)

Reply:
Already [fixed](https://github.com/canonical/traefik-route-k8s-operator/pull/25).
But on github they run in isolation so it probably won't fail..?

Reply:
oh. Yeah I'll revert then. Tomorrow first thing

Reply:
done